pkg/apiserver/api/v1/event.proto

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright 2019 The KubeCarrier Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+syntax = "proto3";
+package kubecarrier.api.v1;
+option go_package = "v1";
+
+import "google/protobuf/any.proto";
+
+message Event {
```

Comment:
Maybe we should also call this `WatchEvent` to align with K8s?
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#watchevent-v1-meta

Reply:
done